services/121-service/src/payments/fsp-integration/intersolve-visa/dto/intersolve-block.dto.ts

```diff
@@ -12,7 +12,7 @@ export enum UnblockReasonEnum {
   UNBLOCK_GENERAL = 'UNBLOCK_GENERAL',
 }
 
-export class IntersolveBlockWalletResponseDto {
+export class IntersolveApiBlockWalletResponseDto {
```

Comment:
AFAIR we agreed that since this is an internal DTO, which sits in the internal/api-intersolve folder, the prefix "Intersolve" is not needed anymore. And the prefix "IntersolveAPI" I do not recall having aligned on.